server/src/test/scala/za/co/absa/atum/server/api/TestData.scala

```diff
@@ -64,18 +64,36 @@ trait TestData {
   protected val measureFromDB1: MeasureFromDB = MeasureFromDB(Some("count1"), Some(Seq("col_A1", "col_B1")))
   protected val measureFromDB2: MeasureFromDB = MeasureFromDB(Some("count2"), Some(Seq("col_A2", "col_B2")))
 
-  // Additional Data
-  protected val additionalDataDTO1: InitialAdditionalDataDTO = Map(
+  // Initial Additional Data
```

Comment:
I think `initial` is not descriptive enough (obviously not just here but also in the model's DTO) 🤔

Reply:
It's only a temporary name so that I can distinguish between newly created types and existing ones. The code related to InitialAdditionalData will be refactored to use the newly created classes. But it's going to be done in a separate PR in order to not affect scope of individual tickets.